examples/plotting/file/line_toggle.py

```diff
@@ -0,0 +1,39 @@
+"""
+Example demonstrating the picking of line objects. The transparency of
+the lines is toggled when clicked.
+"""
+
+import numpy as np
+from bokeh.plotting import output_file, show, figure, Figure
+from bokeh.models import TapTool, CustomJS
+
+# The use of the tap tool causes Bokehjs to "select" a line when it is
+# clicked on, which is not what we want in this example; we want to
+# toggle a visual property of the line ourselves. To work arount this,
+# we modify the selection_glyph and nonselection_glyph as well.
```

Comment:
I just found out that the solution to this problem as done below, is insufficient to realize the kind of behavior that we need in the project. What I would want, is a way to tell Bokeh that I want to know about tap events, but that Bokeh is not going to "select" anything. Thoughts?
cc @bryevdv